carbon-mb/components/transports/mqtt/src/main/java/org/wso2/carbon/andes/transports/mqtt/adaptors/andes/utils/MqttUtils.java

```diff
@@ -198,10 +123,10 @@ public static ByteBuffer getContentFromMetaInformation(AndesContent content)
      * @param messageRoutingKey    The message destination routing key without wildcards
      * @return Is queue bound routing key match the message routing key
      */
-/*    public static boolean isTargetQueueBoundByMatchingToRoutingKey(String queueBoundRoutingKey,
+    public static boolean isTargetQueueBoundByMatchingToRoutingKey(String queueBoundRoutingKey,
                                                                    String messageRoutingKey) {
-       // return SubscriptionsStore.matchTopics(messageRoutingKey, queueBoundRoutingKey);
-    }*/
+        return SubscriptionsStoreUtil.matchTopics(messageRoutingKey, queueBoundRoutingKey);
+    }
```

Comment:
Can we add the method body of SubscriptionsStoreUtil.matchTopics here since it is not used anywhere else without calling a separate method?